python/nav/web/ipdevinfo/views.py

```diff
@@ -108,13 +106,16 @@ def search(request):
     else:
         search_form = SearchForm()
 
-    return render_to_response('ipdevinfo/search.html',
-                              {'errors': errors, 'netboxes': netboxes,
-                               'navpath': NAVPATH, 'query': query,
-                               'title': create_title(titles),
-                               'search_form': search_form},
-                              context_instance=RequestContext(
-                                  request, processors=[search_form_processor]))
```

Comment:
This is not a trivial case. You ought to test, in the shell, if the resulting dict has the same contents.

(You might also check if the search form is used in the template at all.)

Reply:
This holds for each time `search_form_processor` is used, and the eventual goal must be to get rid of it all together or use something like `create_context` further up.

Reply:
I know this is not trivial. I have reviewed all templates and found no references to the items this processor adds in any of the templates, except for one view where this item was already in the context data manually. The commit removes this context_processor altogether

Reply:
Full speed ahead then.